2022-08-29 12:22:10,110 INFO werkzeug MainThread :  * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:5000
 * Running on http://10.10.10.235:5000 (Press CTRL+C to quit)
2022-08-29 12:22:32,782 ERROR app Thread-1 : Exception on /yaml-template/ [POST]
Traceback (most recent call last):
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/flask/app.py", line 2077, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/flask/app.py", line 1525, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/flask/app.py", line 1523, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/flask/app.py", line 1509, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/app/views.py", line 29, in yaml_add
    file = yaml.safe_load(file)
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/yaml/__init__.py", line 125, in safe_load
    return load(stream, SafeLoader)
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/yaml/__init__.py", line 81, in load
    return loader.get_single_data()
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/yaml/constructor.py", line 49, in get_single_data
    node = self.get_single_node()
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/yaml/composer.py", line 36, in get_single_node
    document = self.compose_document()
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/yaml/composer.py", line 55, in compose_document
    node = self.compose_node(None, None)
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/yaml/composer.py", line 84, in compose_node
    node = self.compose_mapping_node(anchor)
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/yaml/composer.py", line 127, in compose_mapping_node
    while not self.check_event(MappingEndEvent):
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/yaml/parser.py", line 98, in check_event
    self.current_event = self.state()
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/yaml/parser.py", line 438, in parse_block_mapping_key
    raise ParserError("while parsing a block mapping", self.marks[-1],
yaml.parser.ParserError: while parsing a block mapping
  in "<unicode string>", line 1, column 1:
    amazon.nodes.Group:
    ^
expected <block end>, but found '<block mapping start>'
  in "<unicode string>", line 75, column 3:
      amazon.nodes.Instance:
      ^
2022-08-29 12:22:32,806 INFO werkzeug Thread-1 : 127.0.0.1 - - [29/Aug/2022 12:22:32] "[35m[1mPOST /yaml-template/?cluster_name=Jupyter_3 HTTP/1.1[0m" 500 -
2022-08-29 12:22:54,622 INFO werkzeug MainThread :  * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:5000
 * Running on http://10.10.10.235:5000 (Press CTRL+C to quit)
2022-08-29 12:22:59,352 ERROR app Thread-5 : Exception on /yaml-template/ [POST]
Traceback (most recent call last):
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/flask/app.py", line 2077, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/flask/app.py", line 1525, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/flask/app.py", line 1523, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/flask/app.py", line 1509, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/app/views.py", line 29, in yaml_add
    file = yaml.safe_load(file)
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/yaml/__init__.py", line 125, in safe_load
    return load(stream, SafeLoader)
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/yaml/__init__.py", line 81, in load
    return loader.get_single_data()
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/yaml/constructor.py", line 49, in get_single_data
    node = self.get_single_node()
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/yaml/composer.py", line 36, in get_single_node
    document = self.compose_document()
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/yaml/composer.py", line 55, in compose_document
    node = self.compose_node(None, None)
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/yaml/composer.py", line 84, in compose_node
    node = self.compose_mapping_node(anchor)
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/yaml/composer.py", line 127, in compose_mapping_node
    while not self.check_event(MappingEndEvent):
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/yaml/parser.py", line 98, in check_event
    self.current_event = self.state()
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/yaml/parser.py", line 438, in parse_block_mapping_key
    raise ParserError("while parsing a block mapping", self.marks[-1],
yaml.parser.ParserError: while parsing a block mapping
  in "<unicode string>", line 1, column 1:
    amazon.nodes.Group:
    ^
expected <block end>, but found '<block mapping start>'
  in "<unicode string>", line 75, column 3:
      amazon.nodes.Instance:
      ^
2022-08-29 12:22:59,356 INFO werkzeug Thread-5 : 127.0.0.1 - - [29/Aug/2022 12:22:59] "[35m[1mPOST /yaml-template/?cluster_name=Jupyter_3 HTTP/1.1[0m" 500 -
2022-08-29 12:23:18,328 ERROR app Thread-6 : Exception on /yaml-template/ [POST]
Traceback (most recent call last):
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/flask/app.py", line 2077, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/flask/app.py", line 1525, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/flask/app.py", line 1523, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/flask/app.py", line 1509, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/app/views.py", line 29, in yaml_add
    file = yaml.safe_load(file)
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/yaml/__init__.py", line 125, in safe_load
    return load(stream, SafeLoader)
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/yaml/__init__.py", line 81, in load
    return loader.get_single_data()
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/yaml/constructor.py", line 49, in get_single_data
    node = self.get_single_node()
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/yaml/composer.py", line 36, in get_single_node
    document = self.compose_document()
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/yaml/composer.py", line 55, in compose_document
    node = self.compose_node(None, None)
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/yaml/composer.py", line 84, in compose_node
    node = self.compose_mapping_node(anchor)
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/yaml/composer.py", line 127, in compose_mapping_node
    while not self.check_event(MappingEndEvent):
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/yaml/parser.py", line 98, in check_event
    self.current_event = self.state()
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/yaml/parser.py", line 438, in parse_block_mapping_key
    raise ParserError("while parsing a block mapping", self.marks[-1],
yaml.parser.ParserError: while parsing a block mapping
  in "<unicode string>", line 1, column 1:
    amazon.nodes.Group:
    ^
expected <block end>, but found '<block mapping start>'
  in "<unicode string>", line 75, column 3:
      amazon.nodes.Instance:
      ^
2022-08-29 12:23:18,330 INFO werkzeug Thread-6 : 127.0.0.1 - - [29/Aug/2022 12:23:18] "[35m[1mPOST /yaml-template/?cluster_name=Jupyter_3 HTTP/1.1[0m" 500 -
2022-08-29 12:23:41,864 ERROR app Thread-25 : Exception on /yaml-template/ [POST]
Traceback (most recent call last):
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/flask/app.py", line 2077, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/flask/app.py", line 1525, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/flask/app.py", line 1523, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/flask/app.py", line 1509, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/app/views.py", line 29, in yaml_add
    file = yaml.safe_load(file)
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/yaml/__init__.py", line 125, in safe_load
    return load(stream, SafeLoader)
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/yaml/__init__.py", line 81, in load
    return loader.get_single_data()
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/yaml/constructor.py", line 49, in get_single_data
    node = self.get_single_node()
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/yaml/composer.py", line 36, in get_single_node
    document = self.compose_document()
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/yaml/composer.py", line 55, in compose_document
    node = self.compose_node(None, None)
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/yaml/composer.py", line 84, in compose_node
    node = self.compose_mapping_node(anchor)
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/yaml/composer.py", line 127, in compose_mapping_node
    while not self.check_event(MappingEndEvent):
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/yaml/parser.py", line 98, in check_event
    self.current_event = self.state()
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/yaml/parser.py", line 438, in parse_block_mapping_key
    raise ParserError("while parsing a block mapping", self.marks[-1],
yaml.parser.ParserError: while parsing a block mapping
  in "<unicode string>", line 1, column 1:
    amazon.nodes.Group:
    ^
expected <block end>, but found '<block mapping start>'
  in "<unicode string>", line 75, column 3:
      amazon.nodes.Instance:
      ^
2022-08-29 12:23:41,865 INFO werkzeug Thread-25 : 127.0.0.1 - - [29/Aug/2022 12:23:41] "[35m[1mPOST /yaml-template/?cluster_name=Jupyter_4 HTTP/1.1[0m" 500 -
2022-08-29 12:24:15,899 INFO werkzeug Thread-26 : 127.0.0.1 - - [29/Aug/2022 12:24:15] "[31m[1mPOST /yaml-template/?cluster_name=Jupyter_3 HTTP/1.1[0m" 400 -
